fast/stages/02-security/main.tf

```diff
@@ -44,4 +44,41 @@ locals {
     "secretmanager.googleapis.com",
     "stackdriver.googleapis.com"
   ]
+  _data_platform_projects = {
```

Comment:
This is not supposed to be here. Stage 2s don't need to know about any of stage3 resources.